.github/workflows/require_label.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: Pull Request Labels
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mheap/github-action-required-labels@v3
+        with:
+          mode: minimum
+          count: 1
+          labels: "documentation, bug, bugfix, chore, enhancement, feature, ignore-for-release, security, sorbet"
```

Comment:
It seems we have a bunch of PRs tagged `feature` when they add new functionality: https://github.com/Shopify/tapioca/pulls?q=is%3Apr+label%3Afeature. I find the distinction from `enhancement` useful.

Removing the `bug` label will be useful though since a lot of PRs are tagged with `bug` but not `bugfix`: https://github.com/Shopify/tapioca/issues?q=label%3Abug.